### PR TITLE
Public Server Update

### PR DESCRIPTION
**Pull request by MichaelModulo**
- Updated server images to PS10.

### CODE REVIEW

File: site/images/server/ps10_banner_text.jpg
Comment (Bloo-dev) [inline site/images/server/ps10_banner_text.jpg:1]: It seems like you never reference this file in code, is this intentional?

Comment (MichaelMiner137) [reply site/images/server/ps10_banner_text.jpg:1]: The images folder has both ps9_banner and ps9_banner_text so I made a version of each for PS10 to be consistent.

Comment (misode) [reply site/images/server/ps10_banner_text.jpg:1]: I'm guessing the one with text was mostly used as promo on socials? I don't think it hurts to have both

Comment (Bloo-dev) [reply site/images/server/ps10_banner_text.jpg:1]: Seems good to me then.

**Review by Bloo-dev — APPROVED** [review]

**Pull request merged**